cmd/juju/charmhub/download.go

```diff
@@ -252,7 +252,8 @@ func (c *downloadCommand) Run(cmdContext *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
-	pb := progress.MakeProgressBar(cmdContext.Stdout)
+	//pb := progress.MakeProgressBar(cmdContext.Stdout)
+	pb := progress.MakeProgressBar(cmdContext.Stderr)
```

Comment:
I'd rather have `--no-progress`, sending on stderr indicates that there is an error, not a back channel for other writes.